pkg/web/templates/partials/sunrise/verification.html

```diff
@@ -1,17 +1,34 @@
-<div id="verify-code" class="mx-auto max-w-4xl">
+<!-- If the confirmation link is valid and hasn't expired, display the email verification -->
```

Comment:
This is how I believe the workflow will work. A user will click a link and htmx will trigger this partial to load. If the confirmation link is valid and hasn't expired users will see the verify email form. If the link has expired, the user see the resend confirmation link view.

Reply:
That makes sense to me! We can either render a different partial in the back-end or use HX-Redirect, but either way, the front-end stays the same.